akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterSharding.scala

```diff
@@ -469,6 +477,11 @@ class ClusterSharding(system: ExtendedActorSystem) extends Extension {
     }
   }
 
+  def defaultShardAllocationStrategy(settings: ClusterShardingSettings): ShardAllocationStrategy = {
+    val threshold = settings.tuningParameters.leastShardAllocationRebalanceThreshold
+    val maxSimultaneousRebalance = settings.tuningParameters.leastShardAllocationMaxSimultaneousRebalance
+    new LeastShardAllocationStrategy(threshold, maxSimultaneousRebalance)
+  }
 }
```

Comment:
Cool, nice to move like this :)

Reply:
good refactoring. Thinking about if it should be public or not. We have the `allocationStrategy ` parameter at a few places so it might be good to be able to create it with this method.